src/App.js

```diff
@@ -1,26 +1,38 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import style from './App.css';
+import { Provider} from 'react-redux';
+import {createStore, applyMiddleware, compose } from 'redux';
+import { restaurantsReducer } from './reducers/Reducers.js';
+import bkgImage  from './restaurant_image.jpg';
+import RestaurantsList from './containers/RestaurantsList.js'
+import thunk from 'redux-thunk';
+
+
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+
+const store = createStore(restaurantsReducer, {
+  isFetching: false,
```

Comment:
So do we still need these defaults?
I think we can drop them, make sure we have an initial state with in the restaurantReducer that should solve issues with "Data/rendering prior to fetching"